pyrms/__init__.py

```diff
@@ -1 +1,18 @@
-import rms
+import os
+import subprocess
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+
+def install():
+    """
+    Install Julia packages required for diffeqpy.
+    """
+    import julia
+    import diffeqpy
+    julia.install()
+    diffeqpy.install()
+    from julia.api import Julia
+    jl = Julia(compiled_modules=False)
+    from julia import Pkg
+    Pkg.add("ReactionMechanismSimulator")
```

Comment:
Can you change this to 
`Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="main"))`

Reply:
We discussed offline and think it's better to have more frequent release of RMS and point the user to a specific version. Although I think `Pkg.add("ReactionMechanismSimulator")` points to a 0.3.4 but not the GitHub release 0.4.0.